Sources/xcproj/XCScheme.swift

```diff
@@ -346,6 +398,9 @@ final public class XCScheme {
             self.useCustomWorkingDirectory = element.attributes["useCustomWorkingDirectory"] == "YES"
             self.debugDocumentVersioning = element.attributes["debugDocumentVersioning"] == "YES"
             self.buildableProductRunnable = try BuildableProductRunnable(element: element["BuildableProductRunnable"])
+            if let commandlineOptions = element.children.first(where: { $0.name == "CommandLineArguments" }) {
```

Comment:
I think AEXMLElement has some query subscripts doesn't it? (I haven't used it in a while)
Something like `if let commandLineOptions = element["CommandLineArguments"]` (no idea if this compiles)